src/quo2/components/drawers/documentation_drawers/view.cljs

```diff
@@ -0,0 +1,33 @@
+(ns quo2.components.drawers.documentation-drawers.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.components.drawers.documentation-drawers.style :as style]
+            [react-native.core :as rn]
+            [quo2.components.buttons.button :as button]))
+
+(defn view
+  "Options
+   - `title` Title text
+   - `cta?` Show CTA button
+   - `cta-label` CTA button label
```

Comment:
consider something simple like "button" in place of `cta`. i.e

`on-press-button`
`button-icon`